webapp/admin_functions.html

```diff
@@ -0,0 +1,65 @@
+<div class="container" ng-init="isRegionCollapsed = !isRegionCollapsed; isUserCollapsed = !isUserCollapsed">
+    <h3>Super Admin Functions</h3>
+    <hr/>
+
+    <div class="panel panel-default">
+        <div class="panel-heading">
+          <h4 class="panel-title">
+            <a ng-click="isRegionCollapsed = !isRegionCollapsed">
+              Regions
+            </a>
+          </h4>
+        </div>
+        <div collapse="isRegionCollapsed" id="regionsCollapsable" collapsed>
```

Comment:
there's some weird thing for me where the cursor when i hover over this is a text cursor instead of a hand cursor, but it's fine to fix this later with the UI update (I think the 'delete' button for tournaments also has this issue)